### DISCUSSION TITLE
A Big Thanks

### DISCUSSION THREAD

**Discussion by ajaybeniwal** [react-hook-form/react-hook-form#3384] [Show and tell]
We were recently using Antd Forms syncing the state with redux  , but recently we got a very complex form requirement  with deep nested arrays  and the form kept hanging with existing architecture , so we decided to make a switch to React Hook Form  useFieldArray was life saver we would not have achieved the functionality in such a short time with any other library out there

**Comment by bluebill1049** [comment]
That's amazing to hear! Thank you for sending such a heartwarming message.
